### PR TITLE
Add unsubscribe option

### PR DESCRIPTION
**Pull request by feliciaan**

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -25,6 +25,8 @@
 #  club_id                 :integer
 #  registration_open       :boolean          default(TRUE)
 #  signature               :text
+#  unsubscribe_option      :boolean
+#  slug                    :string
```
Comment (Silox) [inline app/models/event.rb]: Where is this used for?

File: config/routes.rb [hunk 2]
```diff
@@ -75,6 +75,10 @@
     end
   end
 
+  get '/unsubscribe/:id/:barcode' => 'unsubscribe#show',  as: 'unsubscribe'
```
Comment (Silox) [inline config/routes.rb]: Why does an unsubscribe need a get resource? See below

File: config/routes.rb [hunk 3]
```diff
@@ -75,6 +75,10 @@
...
+
+  delete 'unsubscribe/:id/:barcode' => 'unsubscribe#delete', as: 'unsubscribe_delete'
```
Comment (Silox) [inline config/routes.rb]: I would put these as a post 'unsubscribe' as a registration collection member. The route then becomes /event/event_id/registration/unsubscribe with a barcode as post param.

File: db/schema.rb [hunk 4]
```diff
@@ -104,6 +104,8 @@
     t.integer  "club_id"
     t.boolean  "registration_open",       default: true
     t.text     "signature"
+    t.boolean  "unsubscribe_option"
```
Comment (Silox) [inline db/schema.rb]: I would change the name to something more descriptive, such as "allow_unsubscribing".

File: app/views/registration_mailer/ticket.html.erb [hunk 5]
```diff
@@ -49,6 +49,9 @@
 
   <p>If you have any problems, you can contact us via mail: <%= mail_to @registration.event.contact_email, @registration.event.contact_email %></p>
 
+  <% if @event.unsubscribe_option %>
+    <p>If you cannot come: <%= link_to "uitschrijven", unsubscribe_path(id: @registration.id, barcode: @registration.barcode)  %></p>
```
Comment (Silox) [inline app/views/registration_mailer/ticket.html.erb]: Should be in english, I would also word it as "Unsubscribe if you're unable to attend." with unsubscribe as link.

File: app/controllers/welcome_controller.rb [hunk 6]
```diff
@@ -1,5 +1,7 @@
 class WelcomeController < ApplicationController
   def index
-    redirect_to event_path(Event.first)
+    if Event.first
+      redirect_to event_path(Event.first)
```
Comment (VandorpeDavid) [inline app/controllers/welcome_controller.rb]: `redirect_to Event.first if Event.first` works as well, but you're still querying ´Event.first` twice.

Comment (Silox) [inline app/controllers/welcome_controller.rb]: I wouldn't really care about this, this is something which should be seeded and thus should not error in the app normally. If it does, it's a clear signal to take action.

**Pull request merged**